configure/files/container.rb

```diff
@@ -0,0 +1,9 @@
+Ohai.plugin(:Container) do
+  provides "container"
+
+  collect_data(:default) do
+    if File.exist?("/.dockerenv") || !!(File.read("/proc/1/cgroup") =~ %r[^\d+:\w+:/(lxc|docker)/])
+      container(true)
+    end
+  end
+end
```

Comment:
This is much better than the check I had in place. It is also a great example of an Ohai plugin I'll look to reference in the future. 👏